jvb/src/main/kotlin/org/jitsi/videobridge/xmpp/XmppConnection.kt

```diff
@@ -54,6 +54,8 @@ class XmppConnection : IQListener {
 
     var eventHandler: EventHandler? = null
 
+    val config = XmppClientConnectionConfig.config
```

Comment:
Or this?

Reply:
I thought it made sense conceptually to expose the config *of this XmppConnection*, but looking again it's not necessary. I removed it.